ci/test/00_setup_env_native_valgrind.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export PACKAGES="valgrind clang llvm python3-zmq libevent-dev bsdmainutils libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-test-dev libboost-thread-dev libdb5.3++-dev libminiupnpc-dev libzmq3-dev"
+export USE_VALGRIND=1
+export NO_DEPENDS=1
+export TEST_RUNNER_EXTRA="p2p_segwit.py"  # Only run one test for now. TODO enable all and bump timeouts
+export RUN_FUNCTIONAL_TESTS=true
+export GOAL="install"
+export BITCOIN_CONFIG="--enable-zmq --with-incompatible-bdb --with-gui=no CC=clang CXX=clang++"  # TODO enable GUI
```

Comment:
Should we be testing with this flag `--with-incompatible-bdb` set to true? Is there are reason that we can't use the correct version of berkleydb here?

Reply:
We can use the bdb package from depends or from the Ubuntu package manager. The package manager doesn't come with the bdb version we want, so `--with-incompatible-bdb` is needed. Switching to `./depends` is possible, but shouldn't matter, as we ignore leaks and memory issues in dependencies anyway. They need to be dealt with upstream.